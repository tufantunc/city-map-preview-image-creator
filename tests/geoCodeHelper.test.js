const geoCodeHelper = require('../geoCodeHelper');

describe('geoCodeHelper when called with', () => {
    test('right parameters, should return city geo code', async (done) => {
        const cityGeoCode = await geoCodeHelper('istanbul', 'Turkey', 'tr_TR');
        expect(cityGeoCode).toMatchSnapshot();
        done();
    });
    test('no parameters, should throw error', async (done) => {
        const testError = new Error('Wrong parameters!');
        expect(geoCodeHelper()).rejects.toEqual(testError);
        done();
    });
});