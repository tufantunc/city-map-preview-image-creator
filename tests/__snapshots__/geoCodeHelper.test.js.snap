// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`geoCodeHelper when called with right parameters, should return city geo code 1`] = `
Object {
  "city": "istanbul",
  "country": "Turkey",
  "language": "tr_TR",
  "latitude": "41.008857",
  "longitude": "28.96747",
}
`;
